regression/tests/studio/test_schedule_and_details_settings.py

```diff
@@ -0,0 +1,93 @@
+"""
+Regression tests for Studio's Setting page.
+"""
+from bok_choy.web_app_test import WebAppTest
+
+from regression.pages.studio.login_studio import StudioLogin
+from regression.pages.studio.settings_studio import SettingsPageExtended
+from regression.tests.helpers import LoginHelper, get_course_info
+
+from regression.pages.studio.utils import (
+    get_text
+)
+
+
+class ScheduleAndDetailsTest(WebAppTest):
+    """
+    Tests for Studio's Setting page.
+    """
+    def setUp(self):
+        super(ScheduleAndDetailsTest, self).setUp()
+        self.login_page = StudioLogin(self.browser)
+        LoginHelper.login(self.login_page)
+        self.course_info = get_course_info()
+
+        self.settings_page = SettingsPageExtended(
+            self.browser,
+            self.course_info['org'],
+            self.course_info['number'],
+            self.course_info['run']
+        )
+        self.settings_page.visit()
+
+    def test_file_format(self):
+        """
+        Scenario: Upload course image of a wrong format.
+        Given that I am on the Settings page of the course.
```

Comment:
nit: i'd put in a newline here so that it reads something like:
```
Scenario: Upload course image of wrong format.

Given that I am on the Settings page...
```